### DISCUSSION TITLE
Mirror forcing  landscape mode or added Zoom?

### DISCUSSION THREAD

**Discussion by blawdi** [AndreyPavlenko/Fermata#462] [Ideas]
Hello, Thank you for your wonderful work!

Would it be possible to have a forced landscape mode, as some applications switch the phone to portrait mode, which becomes very difficult to use in landscape mode because you have to press small buttons, or alternatively add a zoom mode?

**Comment by AndreyPavlenko** [comment]
It's already forced, but the apps have full control over the screen orientation. Moreover, some apps do not have support for landscape at all. There is an Xposed hook that forcibly enables landscape for selected apps, but some apps crash once rotated.

**Reply by blawdi** [reply]
Thank you for your response. Indeed, I tested it with an application that forces landscape mode, and it works as I wanted in my case :)

**Reply by tlspruce** [reply]
what is the hook? Is there any available module that forces portrait mode, not landscape mode?
